cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -169,6 +176,34 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
         });
 
         detailRKE2ClusterPage.waitForPage(undefined, 'registration');
+
+        cy.get('.checkbox-label').contains('Insecure:').click();
+
+        createRKE2ClusterPage.commandFromCustomClusterUI().then(($value) => {
+          const registrationCommand = $value.text();
+
+          cy.log(registrationCommand);
```

Comment:
can remove?

Reply:
I will remove the redundant ones like the one in `cluster-create`, but these are helpful to catch errors due to the limited logging of `cy.exec`